template_config.yml

```diff
@@ -40,16 +42,21 @@ pre_job_template: null
 publish_docs_to_pulpprojectdotorg: true
 pulp_scheme: https
 pulp_settings: null
-pulpcore_branch: '3.14'
+pulpcore_branch: '3.15'
```

Comment:
making it 3.16 may solve that CI issue

Reply:
I backed off pulp_file instead, which also works - opinion on which is the "better" thing to do?

Reply:
I think both approaches are fine